packages/xod-client/src/editor/containers/patchModes/linking.jsx

```diff
@@ -0,0 +1,104 @@
+import R from 'ramda';
+import React from 'react';
+import { HotKeys } from 'react-hotkeys';
+
+import { EDITOR_MODE } from '../../constants';
+
+import PatchSVG from '../../../project/components/PatchSVG';
+import * as Layers from '../../../project/components/layers';
+
+import {
```

Comment:
Looks strange, that these utils imported from "selecting" mode.
How about to move it somewhere into "modeUtils.js" or something like this?